README.md

```diff
@@ -157,6 +157,8 @@ Paragon currently uses [the default conventional Angular changelog rules](https:
 2. `fix` (`patch` release)
 3. `perf` (`patch` release)
 
+There are other commit types that will not trigger a release that you can use at your own discretion. Suggested prefixes are `docs`, `chore`, `style`, `refactor`, and `test` for non-changelog related tasks.
```

Comment:
Mind dropping a link in this change to https://github.com/marionebl/commitlint/blob/master/%40commitlint/config-angular-type-enum/index.js#L1-L12 ?

Reply:
Ah, there's the master list! Will do.